ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/ChainSync.hs

```diff
@@ -103,7 +109,9 @@ basicChainSyncClient
       { InFutureCheck.proxyArrival = Proxy
       , InFutureCheck.recordHeaderArrival = \_ -> pure ()
       , InFutureCheck.judgeHeaderArrival = \_ _ _ -> pure ()
-      , InFutureCheck.handleHeaderArrival = \_ -> pure Nothing
+      , InFutureCheck.handleHeaderArrival = \_ ->
```

Comment:
Would you explain this change --- seems unrelated based on the diff I've reviewed so far.